Extensions/fontconfig/src/fontconfig.lisp

```diff
@@ -230,3 +316,23 @@
     (let ((result (fc-config-app-font-add-file (find-config) name)))
       (when (zerop result)
         (error "Unable to add font file: ~s" file)))))
+
+(deftype unicode-codepoint () '(integer 0 #.(* (expt 2 16) 17)))
+
+(defun charset-contains-p (charset char)
```

Comment:
A better name would be `charset-contains-char-p`

Reply:
I agree. This has been address in 039c5c6a73d304ff6b0d8a257fe06e2370a33ba5